examples/js/bots/run_bots.ts

```diff
@@ -1,43 +1,33 @@
 import { MMByPriceBot } from "./mm_external_price_bot";
 import * as regression from "regression";
-import { Account } from "fluidex.js";
-import { defaultRESTClient, RESTClient } from "../RESTClient";
-import { defaultClient as defaultGrpcClient, Client as grpcClient, defaultClient } from "../client";
+import { defaultClient } from "../client";
 import { sleep } from "../util";
-import { ORDER_SIDE_BID, ORDER_SIDE_ASK, ORDER_TYPE_LIMIT, VERBOSE, TestUser } from "../config";
-import {
-  estimateMarketOrderSell,
-  estimateMarketOrderBuy,
-  execMarketOrderAsLimit_Sell,
-  execMarketOrderAsLimit_Buy,
-  rebalance,
-  printBalance,
-  totalBalance,
-} from "./utils";
+import { VERBOSE, TestUser } from "../config";
+import { rebalance, printBalance, totalBalance } from "./utils";
 import { executeOrders } from "./executor";
-import { depositAssets, getPriceOfCoin } from "../exchange_helper";
+import { getPriceOfCoin } from "../exchange_helper";
 
 //const VERBOSE = false;
 console.log({ VERBOSE });
 
-const market = "ETH_USDT";
-const baseCoin = "ETH";
-const quoteCoin = "USDT";
+const market = "BTC_FEE";
+const baseCoin = "BTC";
+const quoteCoin = "FEE";
```

Comment:
Instead of defining variables `market`, `baseCoin` and `quoteCoin` here, use the variables `base`, `quote` and `market` from the file `examples/js/config.ts`.

I recently switched the base and the quote such that now the base is `DIF` and the quote is `BTC`. This should automatically be fixed by using the variables from `config.ts`